configuration/pih/liquibase/liquibase.xml

```diff
@@ -154,4 +154,17 @@
         </sql>
     </changeSet>
 
+    <changeSet id="20210602-delete-unused-ces-programs" author="jmbabazi">
+        <preConditions onFail="MARK_RAN">
+        <sqlCheck expectedResult="0">
+            select count(*) from patient_program where program_id in
+            (select program_id from program where uuid in ("9fe0c32c-981f-4283-b4f4-168285a52c7d", "40657f91-5991-44fe-a16f-220dbeb9dcbd", "1eb13158-f39c-444a-b77c-78f1d78123be"));
+        </sqlCheck>
+        </preConditions>
+        <comment>If the count above is 0, then remove these programs (Hypertension program, Diabetes program, Epilepsy program)</comment>
```

Comment:
We only want to remove these programs if the count of patient programs = 0, not above 0?

(It looks like the code is doing the right thing, but the comment is wrong?)

Reply:
Thanks @mseaton , @mogoodrich . I removed the above from the comments